src/pages/orders/OrderDetails.js

```diff
@@ -105,37 +119,63 @@ const OrderDetails = (props) => {
                 <Col>
                   <div className="d-sm-flex flex-wrap">
                     <h3 className="mb-0 mr-2">Date: </h3>
-                    <label>
-                      {buyer.firstName} {buyer.lastName}
-                    </label>
+                    <label>{order.createdAt.split("T")[0]}</label>
                   </div>
                   <div className="d-sm-flex flex-wrap">
                     <h3 className="mb-0 mr-2">Time: </h3>
-                    <label>{buyer.email}</label>
-                  </div>
-                  <div className="d-sm-flex flex-wrap">
-                    <h3 className="mb-0 mr-2">Total: </h3>
-                    <label>{order.orderAmount * order.quantity}</label>
+                    <label>{order.createdAt.split("T")[1].split(".")[0]}</label>
```

Comment:
Using `split`s in dealing with dates is inefficient. Use the built-in [`Date`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date) API instead. 

<br />

And format of the date should be in `mm/dd/yyyy`.